roles/validations/tasks/edpm/hotfix.yml

```diff
@@ -0,0 +1,83 @@
+# This job tests the functionality of the dataplane-operator to take an arbitrary hotfixed
+# image as input, then role out the hotfix to the deployed nodes.
+#
+# This job was created to satisfy:
+# https://issues.redhat.com/browse/osprh-5326
+
+# Since jobs use different names for their OpenStackDataPlaneNodeSet's, we need to dynamically
+# determine the name of the currently deployed one
+- name: Determine name of deployed NodeSet
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  cifmw.general.ci_script:
+    output_dir: "{{ cifmw_validations_basedir }}/artifacts"
+    script: >-
+      oc get osdpns --no-headers -o custom-columns=":metadata.name"
+  register: deployed_nodeset_name
+
+# Set fact for the nova-compute image we will leverage in this job.
+- name: Set nova-compute image fact
+  ansible.builtin.set_fact:
+    _hotfixed_edpm_nova_compute_image: quay.io/podified-antelope-centos9/openstack-nova-compute:current-podified
```

Comment:
why not a default variable for this?

Reply:
I just didn't envision that this would need to be tunable on a per job basis. If we're running downstream, we'll be using `registry.redhat.io` images, so an execution of this job would result in some upstream image being pulled in which accomplishes the desired goal.

We can add it to the `defaults/main.yml` if you prefer. Just felt less complicated to avoid unnecessary tunable options.

Reply:
I tend to agree with Pablo here, I wouldn't block on it but if you define it in defaults then we skip a set_fact and it keeps a potentially tunable var with the rest. Maybe we need to update the tag in the future idk.

Reply:
Updated now